slides.md

```diff
@@ -4,7 +4,149 @@
+
+TLDR: they're mostly interchangeable, but prefer arrow syntax over function.
+
+Long explanation: The main differences are:
+1. Arrow functions cannot be used as constructors (a function works as a class constructor of itself, kind of, `class` is just syntactical sugar).
+2. `this` in arrow functions is easier to reason about: it's the `this` of the scope in which the function is declared. In a normal function, `this` might change depending on how it's invoked.
+
+```javascript
+function say(name) {
+  console.log(`Hello ${name}`);
+};
+
+const say = (name) => {
+  console.log(`Hello ${name}`);
+};
+
+const double = n => n * 2;
+const makeConfig = () => ({debug: true, log: false}); // returns the object
+```
+
+You can _spread_ objects and arrays, which applies their content to the current expression. The order in which you spread your object **matters**: to decide which ones overrides in case of objects with the same fields, earlier declarations are overridden.
+
+```javascript
+const numbers = [1, 2, 3];
+const add = (a, b, c) => a + b + c;
+const total = add(...numbers); // 6
+const twice = [...numbers, ...numbers]; // [1,2,3,1,2,3]
+
+const defaults = {
+  log: false,
+  debug: true
+}
+
+const options = {
+  debug: false
+}
+
+const overrideOptions = {...options, ...defaults} // {log: false, debug: true}
+const overrideDefaults = {...defaults, ...options} // {log: false, debug: false}
+const overrideWithFields = {...defaults, debug: false} // {log: false, debug: false}
+```
+
+You can **destructure** (access their inner values directly) objects and arrays.
+```javascript
+const [first, second] = [1,2,3]; // first: 1, second: 2
+const [, last] = [1,2,3]; // last: 3
+const add = ([first, second]) => first + second;
+add([1,2]);
+
+const {a, b} = {a: 1, b: 2}; // a: 1, b: 2
+const add = ({first, second}) => first + second;
+add({first: 1, second: 2});
+```
+
+To check for equality, **always** use the triple equal `===`. Objects are **never** equal between each other. `==` casts the two object to the same type before comparing, and assumptions on the behaviour might lead to unexpected results because the implementation might be not intuitive.
+
+```javascript
+const a = 1;
+const b = "1";
+const c = 1;
+a === b // false
+a === c // true
+
+{a: 1} === {a: 1} // false
+```
+
+To declare a class, use the `class` keyword. To reference fields, use the `this` keyword.
+
+```javascript
+class Hello extends World {
+  constructor(options) {
+    super(options);
+    this.name = options.name;
+  }
+
+  speak() { // method declared as a function
+    console.log(`Hi ${this.name}`);
+  }
+
+  fullName = () => this.name + this.surname; // method declared as arrow function
+}
+```
+
+You can use `export` to export a variable/class/function. You can use `import` to import 3rd party libraries installed via npm and local files.
```

Comment:
I'd clarify that "3rd party libraries" might also mean "or another of your source code files in the same project"

Reply:
Hm yeah, better to rephrase this, basically 3rd party libraries will be included from something like `"react-native"`, while local source files will be included with a relative path like `./Hello.js`